convert2rhel/actions/report.py

```diff
@@ -89,6 +89,23 @@ def summary_as_json(results, json_file=CONVERT2RHEL_JSON_RESULTS):
         json.dump(envelope, f)
 
 
+def wrap_paragraphs(text, width=70, **kwargs):
+    """
+    Wrap the paragraphs for a given text respecting the line breaks defined in
+    the string (if any).
+
+    This solution was taken from
+    https://github.com/python/cpython/issues/46167#issuecomment-1093406764,
+    which is a solution to textwrap not properly respecting line breaks inside
+    strings.
+    """
+    return [
+        line
+        for para in text.splitlines()
+        for line in textwrap.wrap(para, width, subsequent_indent="    ", **kwargs) or [""]
+    ]
```

Comment:
(Note: This is coupled with a suggestion/justification in the summary() function where this is called from.)
``` python
def format(text, width, **kwargs):
    output = []
    first = True
    indent = ""
    subsequent_indent = "    "

    for paragraph in text.splitlines():
        for line in textwrap.wrap(paragraph, width, initial_indent=indent,
subsequent_indent=subsequent_indent, **kwargs) or [""]:
            output.append(line)

        if first:
            indent = subsequent_indent
            subsequent_indent = ""
            first = False

    return "\n".join(output)
```

Reply:
copied this over to #888